openmetadata-ui/src/main/resources/ui/src/styles/fonts.less

```diff
@@ -89,6 +89,10 @@ pre {
   }
 }
 
+.text-announcement {
```

Comment:
fonts is generic file, announcement is not color type.
functionality specific colors shouldn't be in these type of standard files.